### DISCUSSION TITLE
Understanding How dbus-serialbattery and dbus-aggregate-batteries Work Together

### DISCUSSION THREAD

**Discussion by James-Lee-007** [Louisvdw/dbus-serialbattery#998] [Q&A]
Hello, I'm seeking assistance in understanding the interaction between the dbus-serialbattery driver and the dbus-aggregate-batteries driver developed by Dr-Gigavolt. I've successfully connected two JK BMSs to my Cerbo GX using dbus-serialbattery and aggregated them using dbus-aggregate-batteries. The aggregated battery is correctly displayed in my device list on Cerbo GX.

However, I'm confused about a few aspects of how these drivers work together and affect the system's configuration:

Configuration Inheritance: The dbus-serialbattery documentation mentioned that to aggregate multiple batteries, one could use additional drivers like dbus-aggregate-batteries. I've done this, but it's unclear whether the configuration changes I've made in the config.ini for dbus-serialbattery apply to the aggregated battery. How does the configuration from dbus-serialbattery interact with the aggregated battery setup?

OWN_CHARGE_PARAMETERS Setting: I noticed the OWN_CHARGE_PARAMETERS option in the settings.py file of the dbus-aggregate-batteries driver. What exactly does setting OWN_CHARGE_PARAMETERS to false do? How does it influence the behavior of the aggregated battery, especially in relation to charge parameters defined elsewhere?

DVCC Page Configuration on Venus OS: When setting up the controlling BMS in the DVCC page of Venus OS, should I select the AggregateBattery created by dbus-aggregate-batteries or one of the individual SerialBatteries (JK BMSs)?

I'd appreciate any guidance or references to further documentation on these points. 

Thank you in advance for your assistance!

**Comment by mr-manuel** [comment]
The config of the dbus-serialbattery driver is only used for the driver itself. Please ask your question in the aggregator repository you are using, since the dbus-serialbattery does not communicate with the aggregator. The aggregator fetches the data from dbus-serialbattery and dbus-serialbattery is not aware that there is an aggregator.
